sections/futures/MarketDetails/useGetMarketData.ts

```diff
@@ -31,6 +33,8 @@ const useGetMarketData = (mobile?: boolean) => {
 
 	const pastRates = useRecoilValue(pastRatesState);
 	const futuresVolumes = useAppSelector(selectMarketVolumes);
+	const marketPrices = useAppSelector(selectMarketPrices);
+	const marketPrice = useAppSelector(selectLatestMarketPrice);
```

Comment:
I would think we display the skew adjusted price here. This is what's displayed as the "mark price" of the market, which would be oracle price adjusted by skew.